### ISSUE TITLE
YourTravel.TV

### ISSUE THREAD

**Issue by AMartinNo1** [simple-icons/simple-icons#3359]
**Name:** YourTravel.TV
**Website:** https://yourtravel.tv/
**Alexa rank:** 1,834,081
  

**Official resources for icon and color:**
https://yourtravel.tv/wp-content/uploads/2019/10/top-logo.svg

**Comment by PeterShaggyNoble** [comment]
Thanks for the request, @AMartinNo1 :+1:

However, per our contributing guidelines, our cut-off Alexa rank for brands is 500k and, while this brand has shown significant growth since you opened your issue, it's popularity does seem to be on the decline again. We'll monitor it for a few days to see if it improves again and stabilises but, in the meantime, is there another metric of popularity we could use to assess its popularity and suitability for inclusion in our library?

**Comment by AMartinNo1** [comment]
Ah, sorry. Busy days. Thanks for reminding me with the label. Not sure if that's enough for you. Anyway, here are a few more stats:

- YouTube: 150k followers (https://www.youtube.com/channel/UC1IRqyZ_LLJKF40PzBgkUww)
- Facebook: 12k followers (https://www.facebook.com/yourtravel.tv)
- Instagram: 49.9k followers (https://www.instagram.com/yourtravel.tv/)

**Comment by PeterShaggyNoble** [comment]
Thanks for the additional info, @AMartinNo1 - 150k YouTube subscribers certainly seems significant enough to me. Would you like to have a read through [our contributing guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md) and give this a try yourself?

**Comment by AMartinNo1** [comment]
Sure, will give it a try 👍🏻

**Comment by AMartinNo1** [comment]
Unfortunately I don't have AI at the moment and I've used my free trial already. So I tried it using Vectonator along with https://jakearchibald.github.io/svgomg/ After downloading it from the SVOMG-Online Tool using precision 3 and default settings I wanted to double check the sizes and such within Vectornator. However, the image is screwed up.

![image](https://user-images.githubusercontent.com/5464339/89109185-4393a880-d43f-11ea-92a2-a3394b735adf.png)

Opening it with chrome it looks fine though. Does following SVG code looks fine to you?

```
<svg xmlns:vectornator="http://vectornator.io" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill-rule="evenodd" clip-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"><path d="M9.501.018L8.98.095 7.601 2.103s-.458.083-.79.177c-.334.094-.509.351-.509.351s.245.198.59.19c.346-.006.81-.06.81-.06l1.903 1.516.519-.077L9.086 2.84s-.222-.281.08-.348a99.144 99.144 0 011.078-.225l.582.462.392-.057-.445-.706.234-.78-.378.054-.448.596s-.788.074-1.098.1c-.311.024-.177-.311-.177-.311L9.5.018zm3.102 1.425c-.428-.01-.875.006-1.335.047l-.134.445.16.271c1.56-.209 3.052-.2 4.29.03 1.44 1.084.367 2.927-2.51 4.21-2.955 1.32-6.703 1.54-8.414.512-.003 0-.004-.003-.004 0-1.22-.772-1.048-2.021.285-3.149-.182.145-.366.297-.522.425-.526.485-.6.57-1.178 1.225-.193.237-.434.57-.616.816-.805 1.163-.678 2.3.556 3.082.005.002.009.004.01.006 2.262 1.4 7.315 1.123 11.279-.642 3.974-1.771 5.375-4.355 3.122-5.772a9.059 9.059 0 00-.917-.482 8.805 8.805 0 00-.716-.314 11.79 11.79 0 00-.76-.261 10.807 10.807 0 00-.78-.214c-.178-.046-.42-.086-.601-.12a8.558 8.558 0 00-1.215-.115zm2.018 7.603l-.987.441 2.803 2.125s.6.432.034.736c-.564.3-2.011 1.037-2.011 1.037l-1.432-.535-.716.318 1.265 1.1.01 1.71.746-.33.492-1.446s1.516-.583 2.118-.803c.604-.215.522.522.522.522l-.298 3.506.99-.441 1.627-4.728s.862-.42 1.468-.79c.607-.368.814-.98.814-.98s-.592-.252-1.272-.047c-.68.203-1.57.562-1.57.562l-4.603-1.957zM.935 10.14c-.27 1.46-.293 1.988-.254 3.239.048.783.502 1.502 1.412 2.07l.007.01c2.177 1.354 6.713 1.646 10.838.72l-.03-1.593-1.135-1.258c-3.738.847-7.41.665-9.452-.538-.007-.008-.024-.016-.03-.024C1.134 12.054.848 11.137.935 10.14zm22.399 2.71c-.058 1.738-2.104 3.717-5.679 5.153-5.274 2.115-11.877 2.24-14.745.28a8.206 8.206 0 001.251 2.095c2.632 1.797 8.692 1.685 13.531-.257 2.833-1.136 4.62-2.646 5.09-4.062l.13-.429c.312-1.08.372-1.98.422-2.78zm-7.455 8.756l-.285.097.73.653s.157.133-.007.204c-.164.07-.582.24-.582.24l-.345-.15-.204.07.284.309-.026.445.214-.074.16-.361s.433-.131.606-.178c.171-.047.13.157.13.157l-.16.964.284-.1.56-1.272s.244-.099.42-.187c.177-.09.248-.251.248-.251s-.157-.086-.35-.044c-.195.041-.446.12-.446.12l-1.231-.642zm-9.868.472c.166.158.689.795 3.33 1.503.714.157 2.509.665 5.828-.07l.02-.305-.26-.412c-3.082.729-6.504.488-8.918-.716z" vectornator:layerName="Untitled"/></svg>
```

**Comment by PeterShaggyNoble** [comment]
@AMartinNo1, [Inkscape](https://inkscape.org/) is a great, free vector editing programme, if you want to give that a try. It also has a few SVGO plugins (I use [this one](https://github.com/konsumer/inkscape-svgo)) so you don't have to go elsewhere to optimise your SVG.

**Comment by AMartinNo1** [comment]
Thanks. Linscape's website looks quite old to me. That's why I did not bother to install it. Anyway, thanks for the response. I will give it a try then 👍🏻

**Comment by AMartinNo1** [comment]
A quick update: The weather forecast says it's gonna be very hot the next days and I don't have an air con ;) So, I most likely won't make any progress.

**Comment by AMartinNo1** [comment]
A quick update: SVG looks fine to me. Will prepare a PR asap 😃

**Comment by AMartinNo1** [comment]
Ok, as I was preparing the PR I noticed the svg is not fully as expected and I don't get used to inkscape (perhaps lack of will 🙈) and I decided to get back on the Adobe Subscription today 😆

**Comment by fbernhart** [comment]
Fixed by #3723
